pipelines/apDetectorVisitQualityCore.yaml

```diff
@@ -105,6 +105,22 @@ tasks:
         nMergedDiaSources: ct
       python: |
         from lsst.analysis.tools.atools import TaskMetadataMetricTool
+  loadDiaCatalogsCore:
+    class: lsst.analysis.tools.tasks.TaskMetadataAnalysisTask
+    config:
+      connections.inputName: loadDiaCatalogs_metadata
+      connections.outputName: loadDiaCatalogs_metadata  # Should be appended with "_metrics"
```

Comment:
Is it really the output the same as the input here?

Reply:
No. `TaskMetadataAnalysisTask` appends `_metrics` to it's output connections, so it will use `loadDiaCatalogs_metadata_metrics`